$OpenBSD: patch-configure,v 1.7 2014/03/26 08:55:26 sthen Exp $

Try to avoid linking against libintl.a, fix version number in paths.

--- configure.orig	Thu Oct 31 23:07:46 2013
+++ configure	Wed Mar 26 08:54:44 2014
@@ -14729,12 +14729,12 @@ else
 #include <libintl.h>
 $gt_revision_test_code
 extern int _nl_msg_cat_cntr;
-extern int *_nl_domain_bindings;
+extern int libintl_nl_domain_bindings;
 int
 main ()
 {
 bindtextdomain ("", "");
-return * gettext ("")$gt_expression_test_code + _nl_msg_cat_cntr + *_nl_domain_bindings
+return * gettext ("")$gt_expression_test_code + _nl_msg_cat_cntr + libintl_nl_domain_bindings
   ;
   return 0;
 }
@@ -15604,7 +15604,7 @@ $as_echo "#define HAVE_DCGETTEXT 1" >>confdefs.h
   fi
 
 
-
+    LIBINTL="$LTLIBINTL"
     INTLLIBS="$LIBINTL"
 
 
